<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scannerClass</class>
 <widget class="QWidget" name="scannerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>1088</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scanner</string>
  </property>
  <widget class="QPushButton" name="btnCloseCam">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CloseCam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenCam">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenCam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartSnap">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>150</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>StartSnap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStopSnap">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>StopSnap</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCamStatus">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CamStatus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDLPConnect">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>530</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ConnectDLP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDLP4500Status">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>840</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>DLP4500Status</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>640</y>
     <width>216</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>FirmwareTag：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFirmware">
      <property name="text">
       <string>xxx</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBoxSLMode">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>590</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PatternSequence</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_PatSeqPatPeriod">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>640</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_PatSeqPatExpTime">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>610</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>657</x>
     <y>640</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>投影仪周期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>610</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>投影仪曝光</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProject">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>690</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>投影</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>730</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>条纹类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxPatternType">
      <item>
       <property name="text">
        <string>纵条纹</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>对焦</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>亮度显示</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CamExpTime">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStopPro">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>690</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>停止投影</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>相机曝光：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>591</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string>Pic</string>
   </property>
  </widget>
  <widget class="QVTKWidget" name="qvtkWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>611</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>1030</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="scanner.qrc"/>
 </resources>
 <connections/>
</ui>
